### ISSUE TITLE
[Bug]: The results generated by depth_leres are inconsistent with the webui renderings, and the renderings for api calls are particularly poor

### ISSUE THREAD

**Issue by RunXiaoLi** [Mikubill/sd-webui-controlnet#1395]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

The results generated by depth_leres are inconsistent with the webui renderings, and the renderings for api calls are particularly poor

### Steps to reproduce the problem

{
    "alwayson_scripts": {
        "ControlNet": {
            "args": [
                {
                    "control_mode": 0,
                    "input_image": "",
                    "lowvram": false,
                    "model": "control_v11f1p_sd15_depth [cfd03158]",
                    "module": "depth_leres",
                    "processor_res": 512,
                    "weight": 1,
                    "enable": true
                }
            ]
        }
    },
    "batch_size": 1,
    "height": 449,
    "resize_mode": 0,
    "seed": -1,
    "steps": 20,
    "width": 800
}

### What should have happened?

What's wrong with parameters

### Commit where the problem happens

webui: 
controlnet: 

### What browsers do you use to access the UI ?

Google Chrome, Microsoft Edge

### Command Line Arguments

```Shell
{
    "alwayson_scripts": {
        "ControlNet": {
            "args": [
                {
                    "control_mode": 0,
                    "input_image": "",
                    "lowvram": false,
                    "model": "control_v11f1p_sd15_depth [cfd03158]",
                    "module": "depth_leres",
                    "processor_res": 512,
                    "weight": 1,
                    "enable": true
                }
            ]
        }
    },
    "batch_size": 1,
    "height": 449,
    "resize_mode": 0,
    "seed": -1,
    "steps": 20,
    "width": 800
}
```

### List of enabled extensions

There is no

 

### Console logs

```Shell
Model loaded in 6.4s (load weights from disk: 0.2s, create model: 0.5s, apply weights to model: 2.1s, apply half(): 2.2s, move model to device: 1.2s).
Guess Mode is removed since 1.1.136. Please use Control Mode instead.
Guess Mode is removed since 1.1.136. Please use Control Mode instead.
Loading model: control_v11f1p_sd15_depth [cfd03158]
Loaded state_dict from [/root/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_v11f1p_sd15_depth.pth]
Loading config: /root/stable-diffusion-webui/extensions/sd-webui-controlnet/models/control_v11f1p_sd15_depth.yaml
ControlNet model control_v11f1p_sd15_depth [cfd03158] loaded.
Loading preprocessor: depth_leres
preprocessor resolution = 512
Calling preprocessor depth_leres outside of cache.
```

**Comment by oHuflying** [comment]
my problem is same, depth leres or ++, canny api result all very bad, webui result is correct.

**Comment by ljleb** [comment]
Make sure to set the resolution through thresholds a and b in the api, their default value is 64.

**Comment by VictorBell** [comment]
same problem, ljleb`s advice didn`t work
